Ginger/GingerCoreCommon/ReporterLib/UserMsgsPool.cs

```diff
@@ -247,7 +247,8 @@ public static void LoadUserMsgsPool()
             Reporter.UserMsgsPool.Add(eUserMsgKey.RunsetNotFoundForLoading, new UserMsg(eUserMsgType.ERROR, "RunSet Not Found", "No runset details were found.", eUserMsgOption.OK, eUserMsgSelection.OK));
             Reporter.UserMsgsPool.Add(eUserMsgKey.RunSetLoadFromReportError, new UserMsg(eUserMsgType.ERROR, "Unable To Load RunSet", "{0}", eUserMsgOption.OK, eUserMsgSelection.OK));
             Reporter.UserMsgsPool.Add(eUserMsgKey.PublishApplicationToOtherEnv, new UserMsg(eUserMsgType.QUESTION, "Publish Application to other Environment", "Do you want to publish this application to all the other existing environments?", eUserMsgOption.YesNo, eUserMsgSelection.No));
-            Reporter.UserMsgsPool.Add(eUserMsgKey.NoApplicationPlatformLeft, new UserMsg(eUserMsgType.INFO, "No Application is available to add", "All the applications are already available in {0} Environment. Please add new application in the Configurations-> Target application page to add application here", eUserMsgOption.OK, eUserMsgSelection.None));
+
+            Reporter.UserMsgsPool.Add(eUserMsgKey.NoApplicationPlatformLeft, new UserMsg(eUserMsgType.INFO, "No Application is available to add", "All the applications are already available in {0} Environment. \nPlease add new application in the Configurations-> Target application page to add application here", eUserMsgOption.OK, eUserMsgSelection.None));
```

Comment:
Use the Ginger resource dictionary instead of Target Application

Reply:
ok Implemented the requested changes